### PR TITLE
add IndexedDBShim@2.2.1 w/ git auto-update

### LINKED ISSUE #10261: [Request] Add indexeddbshim
**Library name:** indexeddbshim
**Git repository url:**  https://github.com/axemclion/IndexedDBShim
**npm package url(optional):**  indexeddbshim
**License(s):** Apache-2.0 
**Official homepage:**  https://github.com/axemclion/IndexedDBShim
**Wanna say something? Leave message here:** Why it still isn't here? Strange. Thank you.

**The lib maps requests to IndexedDB to WebSQL for browsers like safari (with buggy IDB implementation) and those, that doesn't support IDB natively at all.**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by lcd78706**
close #10261, cc @brettz9

Pull request for issue: #10261 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/axemclion/IndexedDBShim
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/indexeddbshim
 * License and its reference:  MIT OR Apache-2.0  https://github.com/axemclion/IndexedDBShim/blob/master/LICENSE-APACHE
https://github.com/axemclion/IndexedDBShim/blob/master/LICENSE-MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 59
   - Count of stars: 736
   - Count of forks: 167
 * NPM download stats (optional):
   - Downloads in the last day: 242
   - Downloads in the last week: 1393
   - Downloads in the last month: 5468

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by lcd78706** [comment]
@x09326  Please review this PR. Thanks!

File: ajax/libs/indexeddbshim/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "type": "git",
+    "url": "git://github.com/axemclion/IndexedDBShim.git"
+  },
+  "keywords": [
+    "indexedDB",
+    "database",
+    "polyfill",
+    "websql"
+  ],
+  "license": "Apache-2.0 or MIT",
```
Comment (maruilian11) [inline ajax/libs/indexeddbshim/package.json]: Please use an array.

**Comment by maruilian11** [comment]
Please don't mention `close issue and cc the author` in comments next time.

File: ajax/libs/indexeddbshim/package.json (hunk 1 above)
Comment (brettz9) [inline ajax/libs/indexeddbshim/package.json]: Not sure what you're doing internally, but an array is not valid for `package.json`: https://docs.npmjs.com/files/package.json#license 

    Some old packages used license objects or a "licenses" property containing an array of license objects:

    Those styles are now deprecated. Instead, use SPDX expressions...

Comment (lcd78706) [inline ajax/libs/indexeddbshim/package.json]: The other files are in master branch, not in the latest version.

Comment (lcd78706) [inline ajax/libs/indexeddbshim/package.json]: @maruilian11  I have modified the part of license. Please review it again.

Comment (maruilian11) [inline ajax/libs/indexeddbshim/package.json]: thanks @brettz9, we use SPDX expressions, too. Just keep the same structure when we meet multi-license lib in cdnjs.

**Comment by maruilian11** [comment]
LGTM

**Comment by maruilian11** [comment]
@x09326 @kennynaoh Please help me make the double check, thanks!

File: ajax/libs/indexeddbshim/package.json (hunk 1 above)
Comment (brettz9) [inline ajax/libs/indexeddbshim/package.json]: FWIW, we've since added parentheses for the license field as per the SPDX specs (on 1/27: )...

**Comment by x09326** [comment]
@lcd78706 
Please rebase to the latest master branch, thanks.

**Comment by lcd78706** [comment]
@x09326  I have rebased it to the latest master branch. Please review it again. Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Let's use its repo name `IndexedDBShim` as the easier to be understood name, any reason you need to change it to `indexeddbshim`?

**Comment by lcd78706** [comment]
@PeterDaveHello 
The name written in package.json is originally indexeddbshim.
I just keep it and didn't change it.

**Comment by PeterDaveHello** [comment]
@lcd78706 We don't decide the library name by a single factor, repository name, readability should also be considered, let's update it.

**Comment by lcd78706** [comment]
@x09326 
I have modified it. Please review it again. Thanks!

**Comment by x09326** [comment]
LGTM
@cdnjs/intern2 please help me review it, thanks.

**Pull request merged**